pool/pgpoolstorage/pgpoolstorage.go

```diff
@@ -414,6 +414,21 @@ func (p *PostgresPoolStorage) SetGasPrice(ctx context.Context, gasPrice uint64)
 	return nil
 }
 
+// DeleteGasPricesHistory deletes all gas prices except the last one
+func (p *PostgresPoolStorage) DeleteGasPricesHistory(ctx context.Context) error {
+	sql := `DELETE FROM pool.gas_price
+		WHERE item_id NOT IN (
+			SELECT item_id
+			FROM pool.gas_price
+			ORDER BY item_id DESC
+			LIMIT 1
+		)`
```

Comment:
Should I always keep the last gas_price as it is currently done in the current sql request?

The field added in the Config ‘CleanHistoryPeriod’ could be renamed as ‘TimeRetention’ to be used for both the trigger period and the sql request.
We end up with:
Every X min, delete records older than X min.

Reply:
Added new commit with new config field.
It deletes rows older than now - x except the most recent because the default price suggester never set new gas price.